### PR TITLE
Update desktop searchbar colors

### LINKED ISSUE #1879: Update background color for the search bar
## Description

The [new header search bar](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8z/Openverse-Design-Library?node-id=2960%3A8190) is darker when not focused and becomes white when focused to type. This is the opposite of the current design and must be updated.
Header content

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Fixes #1879 by @dhruvkb

## Description

This PR updates the searchbar colors to match the desktop mockups, and adds the VR tests.

## Testing Instructions

The searchbar on the homepage and on the search pages on the desktop widths should match the mockups.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by zackkrida — CHANGES_REQUESTED** [review]
Looks great, one small error. The _hover_ state is also meant to be the gray background, and the text darkens a bit:

https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?node-id=2960%3A8723

image

**Comment by obulat** [comment]
> Looks great, one small error. The _hover_ state is also meant to be the gray background, and the text darkens a bit:

I tried changing the hover state to match the mockups you linked, @zackkrida, but then I noticed that they are not updated: the [resting](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?node-id=352%3A537) state is white, and the hover and active states are darker; the color they use is still `dark-charcoal-06` while the new mockups use `dark-charcoal-10`. I think it's better to leave it the way it was in this PR and add changes after Francisco is back.

File: src/components/VContentSwitcherOld/VSearchTypeButtonOld.vue [hunk 1]
```diff
@@ -5,7 +5,8 @@
       sizeClasses,
       {
         'max-w-[10rem] sm:max-w-[20rem] md:max-w-[16rem]': isHeaderScrolled,
```
Comment (obulat) [inline src/components/VContentSwitcherOld/VSearchTypeButtonOld.vue:7]: This change is not related to the desktop searchbar, but it fixes the homepage: when the desktop search type button is focused, it should be white and should not have a border (it has a pink ring instead).

**Comment by obulat** [comment]
> This is great, matches the mockups!
> 
> Would using a flag to differentiate the styles have been easier than splitting the input field into old and new components? That way we wouldn't have to update all the various references to the component.

I feel like it's much easier to safely develop both old and new components when they are separated than to use the flag for styles. It is very easy to accidentally break both :(

**Pull request merged**